data/human/human missions.txt

```diff
@@ -4965,3 +4965,184 @@ mission "Riley"
+			label front-mountain
+			`	Liam, with the help of the map, guides all of you to a flat wall. He looks a bit embarrassed as he says, "There's a phrase our father left in the instructions: 'Edr- i fen'. I do not know what it means, but I'm certain it has something to do with one of these fantasy novels he used to read."`
+			choice
+				`	"Sounds funny, but let's say it."`
+				`	"You've got a strange dad, but I like his sense of humor."`
+			`	Sophie counts to three, and then all of you shout out the phrase together.`
+			`	Suddenly, a bright blue line appears on the wall. For a second, a flash blinds you and there is a dim hissing noise, like the sound of rock being cut by a laser. As you look back, the line forms into a blue rectangle, the shape rapidly expanding down the wall. When it reaches the ground, the shape has become large enough to cover a human.`
+			`	For a moment, nothing happens, and the siblings are too stunned to do anything. The blue rectangle starts flashing between different hues of blue, until the whole rectangle and the stone it covered disappears suddenly to leave behind a cavity in the wall. "I could have sworn the stone was real," Liam says under his breath.`
+			choice
+				`	"Well, this is a real adventure. Lets go!"`
+						goto enter
+				`	"We don't know what's inside there. Be careful."`
+			`	"Well, now that you say it, I recently bought a small camera drone," Liam responds. "We can use it to scout out the hallway. It's only in infrared, so we won't see too many details," he adds. Liam starts the drone and guides it using a small remote control with integrated display. After some time, the drone flies back out and he tells you, "I think it is safe. We can go."`
+			label enter
+			`	The three of you enter the hallway behind the opening in single file. Strangely, the stone walls are perfectly smoothed off, and the light that guides you through seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "That's totally our dad."`
+			`	For a long while, the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room, brilliantly bathed in blue light. On the walls, there are all kinds of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
+			choice
+				`	"(Go up to the sword.)"`
+						goto sword-scene
+				`	"(Take a look around first.)"`
+			`	You walk around, looking at the various alien artifacts. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
```

Comment:
```suggestion
			`	You walk around to the right of the dome to a collection of various alien artifacts, all steel in color and rather blocky. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
```